src/vm/hints/secp/field_utils.rs

```diff
@@ -553,4 +636,360 @@ mod tests {
             ))
         );
     }
+
+    #[test]
+    fn run_is_zero_pack_ok() {
+        let hint_code = "from starkware.cairo.common.cairo_secp.secp_utils import SECP_P, pack\n\nx = pack(ids.x, PRIME) % SECP_P".as_bytes();
+        let mut vm = VirtualMachine::new(
+            BigInt::new(Sign::Plus, vec![1, 0, 0, 0, 0, 0, 17, 134217728]),
```

Comment:
We can replace this with a module level `bigint_str` for legibility and reusability.

Reply:
True,
I created a `vm_prime!()` macro, tell me what you think
https://github.com/lambdaclass/cleopatra_cairo/pull/332/commits/216a6e148595ea1545f5c376b6f1fdc564865c38